CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ BUG FIXES:
 IMPROVEMENTS:
 * Helm:
   * API Gateway: Set primary datacenter flag when deploying controller into secondary datacenter with federation enabled [[GH-1511](https://github.com/hashicorp/consul-k8s/pull/1511)]
+  * API Gateway: Allow controller to create and update Secrets in order to support distroless Envoy images [[GH-1542](https://github.com/hashicorp/consul-k8s/pull/1542)]
```

Comment:
Considering something more like this. Thoughts?

```suggestion
  * API Gateway: Allow controller to create and update Secrets for storing Consul CA cert alongside gateway Deployments [[GH-1542](https://github.com/hashicorp/consul-k8s/pull/1542)]
```

Reply:
Looks good!